slycot/src/AB08NX.f

```diff
@@ -199,8 +199,8 @@ SUBROUTINE AB08NX( N, M, P, RO, SIGMA, SVLMAX, ABCD, LDABCD,
       INTEGER           ILAENV
       EXTERNAL          ILAENV
 C     .. External Subroutines ..
-      EXTERNAL          DLAPMT, DLARFG, DLASET, DLATZM, DORMQR, DORMRQ,
-     $                  MB03OY, MB03PY, XERBLA
+      EXTERNAL          DLAPMT, DLARFG, DLASET, SLCT_DLATZM, DORMQR,
```

Comment:
OK then, our machine had an extension to its FORTRAN 77 then. I actually sold my FORTRAN book to some Indonesian visitors years ago ....